stdlib/map.gr

```diff
@@ -8,115 +8,106 @@ import WasmI32 from "runtime/unsafe/wasmi32"
 import { allocateArray } from "runtime/dataStructures"
 
 // TODO: Consider implementing this as List<(Box<k>, Box<v>)>
-record Bucket<k, v> {
-  mut key: k,
-  mut value: v,
-  mut next: Option<Bucket<k, v>>
-}
+record Bucket<k, v> { mut key: k, mut value: v, mut next: Option<Bucket<k, v>> }
 
-record Map<k, v> {
-  mut size: Number,
-  mut buckets: Array<Option<Bucket<k, v>>>
-}
+record Map<k, v> { mut size: Number, mut buckets: Array<Option<Bucket<k, v>>> }
 
 // TODO: This could take an `eq` function to custom comparisons
-export let makeSized = (size) => {
-  let buckets = Array.make(size, None);
-  {
-    size: 0,
-    buckets
-  }
+export let makeSized = size => {
```

Comment:
Maybe records should always break over multiple lines rather than be condensed onto one line?

Reply:
Like @ospencer said in chat, it's probably better to keep them on one line unless they wrap from being too long.